src/bitcoin/payment.rs

```diff
@@ -57,9 +64,14 @@ pub async fn create_payjoin(
     let original_psbt = sign_original_psbt(wallet, psbt).await?;
     info!("Original PSBT successfully signed");
 
-    // TODO use fee_rate
-    let pj_params = payjoin::sender::Configuration::non_incentivizing();
-    let (req, ctx) = pj_uri.create_pj_request(original_psbt, pj_params)?;
+    let pj_params = match fee_rate {
+        Some(fee_rate) => {
+            payjoin::send::Configuration::min_fee_rate_sat_per_vb(self, fee_rate.as_sat_per_vb())
+        }
+        None => payjoin::send::Configuration::non_incentivizing(),
```

Comment:
Some(fee_rate) will enforce that the receiver provides enough fees to at least match the sender's preferred feerate. It doesn't yet allow the receiver to spend some from sender's change to cover that feerate (which imo it could for up to 1 additional input since that is a worthwhile spend for privacy). None sets no such requirement nor offers a fee budget to the receiver.